### ISSUE TITLE
Create a view in the UI to show completed/queued digital bookplate batches

### ISSUE THREAD

**Issue by shelleydoljack** [sul-dlss/libsys-airflow#1255]
Blocked by #1254 
When a user triggers the add 979 field DAG, they should be sent to a page that displays their batch in the queue. We want a page to show batches in the queue and batches completed. Here is a screenshot of completed batches from our previous UI:
![Screenshot 2024-10-08 at 11 24 36 AM](https://github.com/user-attachments/assets/a91f087c-1bb4-44c7-8793-cbf223995f1f)

The completed/queue is either driven by the status of the dag_run or a value in the completed_date column in the table.

**Comment by shelleydoljack** [comment]
As an airflow admin user, I can see the status of the dag runs. As a user who has access to upload files to the UI, I do not have the ability to view the DAGs and the statuses of their runs. This view would be explicitly for those users to understand what has happened to their file once submitted. It would also be for the airflow admin user to understand who submitted what when (although we could look for an email of the submission). It is a "nice to have" and is not necessary for the work cycle.

**Comment by shelleydoljack** [comment]
This isn't needed because airflow will show all the scheduled and queued dag runs. If users start asking for something to track their add 979s, then we can revisit this.
